Tests/Fixtures/TestProject/Project.xcodeproj/project.pbxproj

```diff
@@ -26,6 +26,8 @@
 		BF_130062884703 /* Alamofire.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = FR_257516580010 /* Alamofire.framework */; };
 		BF_138356261076 /* InterfaceController.swift in Sources */ = {isa = PBXBuildFile; fileRef = FR_363921640403 /* InterfaceController.swift */; };
 		BF_144945303317 /* Alamofire.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = FR_410645050443 /* Alamofire.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
+		BF_151769972761 /* SomeFramework.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = FR_654302359039 /* SomeFramework.framework */; };
+		"BF_151769972761-1" /* SomeFramework.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = FR_654302359039 /* SomeFramework.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```

Comment:
@yonaskolb Do you know what causes this?

Reply:
Hmm I don't, interesting. Looks like an issue with `.framework` dependency embedding. We should get it fixed.
As an explanation for the id, if a duplicate is generated it will have a `-number` appended. The addition of a `-` also surrounds it quotes which as a plus, makes it standout.